src/utils/dates.js

```diff
@@ -10,3 +10,12 @@ const ONE_DAY_IN_MILLISECONDS = 86400000;
 export function getFutureDate(offset) {
 	return new Date(Date.now() + offset * ONE_DAY_IN_MILLISECONDS);
 }
+
+export function getDaysBetweenDates(lastTransaction) {
+	let today = new Date();
+	// get both dates in milliseconds
+	let timeDifference = today.getTime() - lastTransaction.toMillis();
```

Comment:
wow, at first I was really confused because I had never heard of .toMillis() and couldn't find it when searching "toMillis() JS" but it's a firebase method, cool! I wanted to make sure getTime and toMillis were tracking the same value (milliseconds since 1970) and they are ⭐

Reply:
Amazing!